Hogajama/hogajama-rs/src/main/java/com/gepardec/hogarama/service/HogaramaServiceConfiguration.java

```diff
@@ -0,0 +1,13 @@
+package com.gepardec.hogarama.service;
+
+public class HogaramaServiceConfiguration {
+
+    public boolean useKafkaWatering() {
```

Comment:
I think this methods could be static. An Injection of this class in WateringKafkaEndpoint.java and WateringMDB.java would be obsolete.

Reply:
IMHO, as of right now, making it static is a valid option. If we want to use other configurations, we could refactor it (yagni).

Reply:
Why refactor later when we can do the interface right the first time? A configuration object is not static in nature.